pkg/app/app.go

```diff
@@ -33,7 +33,7 @@ func NewCR(name, appName, appVersion, appCatalog string) *applicationv1alpha1.Ap
 				InCluster: true,
 			},
 			Name:      appName,
-			Namespace: "giantswarm",
+			Namespace: appNamespace,
```

Comment:
Changing this is the easiest part I'd say. You'll have to change architect and probably architect-orb.

- https://github.com/giantswarm/architect/tree/master/cmd/create/appcr
- https://github.com/giantswarm/architect-orb/blob/0872360cb8c5d7ff6a39fed40533141be08b3818/src/commands/push-to-app-collection.yaml#L27

Reply:
I know, I have all PRs ready. Waiting for approve

Reply:
If we could flip `appName, appNamespace` to `appNamespace, appName` that would make my OCD happy. This is how it's done in K8s resource paths.